content/en/developers/authorization/oauth2_endpoints.md

```diff
@@ -0,0 +1,234 @@
+
+#### Overview
+When a user clicks the **Authorize** button on the consent page, a `POST` request is automatically sent to the [authorization endpoint][2] to verify the request and return a unique authorization code. The user is redirected to your application’s `redirect_uri` with the authorization code parameter in the query component.
+
+#### Request 
+Your application does not need to make this authorization request. This step is a response to the previous user authorization request and is automatically requested by Datadog when a user successfully authorizes an application. 
+
+
+[1]: https://datatracker.ietf.org/doc/html/rfc6749#section-4.1.2.1
+[2]: https://datatracker.ietf.org/doc/html/rfc6749#section-3.1
+{{% /tab %}}
+{{% tab "Token Endpoints" %}}
+
+## Exchange authorization code for access token
+
+### `POST /oauth2/v1/token`
+
+#### Overview
+
+Once an authorization code is returned from the authorization request, your application can exchange this code for an access token and a refresh token. The authorization code is extracted from the redirect URI and sent in a `POST` request to Datadog’s OAuth2 [token endpoint][1]. 
+
+Datadog [access tokens][2] are short-lived tokens with a time-to-live (TTL) of 1 hour that grant access to Datadog APIs. [Refresh tokens][3] for Marketplace OAuth clients are long-lived tokens with no expiration (TTL of infinity) that are used to automatically obtain a new access token each time it expires. When a user revokes their authorization, they have to re-authorize a new set of access and refresh tokens for the application (the refresh token expires). 
+
+#### Request
+
+The [access token request][4] is made with the following parameters in the body of the `POST` request with the `application/x-www-form-urlencoded` format:
+
+|  HTTP Body Parameter               | Description                                                                                                                                                                                        |
+|------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `redirect_uri`                       | The same [redirection endpoint][5] sent in the authorization requests.                                                                                                                                   |
+| `client_id`                          | The client ID of your OAuth2 client.                                                                                                                                                                |
+| `client_secret` (if issued)          | The client secret of your OAuth2 private client.                                                                                                                                                    |
```

Comment:
I'm thinking maybe we should remove the word private here. It might be confusing since we talk about private vs. published client, but this I think is referring to `confidential` client. If that is the case, we can switch it to `confidential`

Reply:
Will update to confidential